abstractions/java/lib/build.gradle

```diff
@@ -42,7 +42,7 @@ publishing {
     publications {
         gpr(MavenPublication) {
             artifactId 'kiota-abstractions'
-            version '1.0.3'
+            version '1.0.4'
```

Comment:
should these be version 0 since nothing's been released yet.

Reply:
right now we're only publishing to the local feed [here](https://github.com/orgs/microsoft/packages?repo_name=kiota) I'm bumping the versions on changes in case people are taking dependencies on it.
When we publish to the "public feeds" we can always reset the version numbers. And we might want to change the names too.

- typescript: @microsoft/kiota-abstrations (I'm happy with that)
- dotnet: KiotaAbstractions, in retrospect I'm not super happy with that. I think it should at least be Kiota.Abstrations or even Microsoft.Kiota.Abstractions. thoughts?
- java: com.microsoft.kiota:kiota-abstractions (I'm happy with that too for now)

Reply:
The .net should probably be microsoft.kiota.abstractions to match the other too. It does mean we will need to sign the build